packages/react-instantsearch-core/CHANGELOG.md

```diff
@@ -1297,7 +1297,7 @@ this will go down afterwards, but for now there's some more duplication
 Previously there were three options to pass a search client: searchClient, appId & apiKey, algoliaClient. The latter two have been removed, and now only `searchClient` is accepted. This searchClient is an instance of the `algoliasearch` module:
 
 ```js
-import algoliasearch from 'algoliasearch/lite';
+import { liteClient as algoliasearch } from 'algoliasearch/lite';
```

Comment:
```suggestion
import algoliasearch from 'algoliasearch/lite';
```

I think this one can stay as-is

Reply:
Yeah wasn't sure about the Changelog ones, I'm fine with reverting them.